packages/wrangler/src/publish.ts

```diff
@@ -327,15 +337,13 @@ export default async function publish(props: Props): Promise<void> {
     if (triggers && triggers.length) {
       deployments.push(
         fetchResult(`${workerUrl}/schedules`, {
-          // TODO: Unlike routes, this endpoint does not support PATCH.
-          // So technically, this will override any previous schedules.
-          // We should change the endpoint to support PATCH.
```

Comment:
Is this in the works, or no longer doable?

Reply:
Using patch is weird because it means the source of truth is now coming from two places. But of course it means you can edit it in the dashboard. Each option has tradeoffs, but this is the one we're picking for now and can revisit later based on feedback.